solving lint issues. (#3475)
webpack/webpack.js.org: src/content/concepts/entry-points.md

### Separate App and Vendor Entries

__webpack.config.js__

```javascript
module.exports = {
  entry: {
    main: './src/app.js',
    vendor: './src/vendor.js'
  }
};
```

__webpack.prod.js__

```javascript
module.exports = {
  output: {

>     filename: [name].contentHash.bundle.js

Reviewer: Please use valid JavaScript.

Author: filename: [name].[contentHash].bundle.js
This fails travis CI test. @montogeek

>     filename: [name].contentHash.bundle.js

Reviewer 2: It lacks single quotes no?
```suggestion
    filename: '[name].contentHash.bundle.js'
```

>     filename: '[name].contentHash.bundle.js'

Reviewer 2: ```suggestion
    filename: '[name].[contentHash].bundle.js'
```

  }
};
```

__webpack.dev.js__

```javascript
module.exports = {
  entry: {

>     filename: [name].bundle.js

Reviewer 2: ```suggestion
    filename: '[name].bundle.js'
```

  }
};
```

__What does this do?__ We are telling webpack that we would like 2 separate entry points (like the above example).

> __Why?__ With this you can import required libraries or files that aren't modified (ex bootstrap, jquery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content Hash remains the same, which allows the browser to cache separately thereby reducing load time.

Reviewer: ```suggestion
__Why?__ With this you can import required libraries or files that aren't modified (i.e. Bootstrap, jQuery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content hash remains the same, which allows the browser to cache them separately thereby reducing load time.
```

Reviewer 2: @montogeek did you mean `e.g.` and not `i.e.` there, bootsrap and jquery are given as an example and not describing the statement in other words

> __Why?__ With this you can import required libraries or files that aren't modified (i.e. Bootstrap, jQuery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content hash remains the same, which allows the browser to cache them separately thereby reducing load time.

Reviewer 2: ```suggestion
__Why?__ With this you can import required libraries or files that aren't modified (e.g. Bootstrap, jQuery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content hash remains the same, which allows the browser to cache them separately thereby reducing load time.
```

> __Why?__ With this you can import required libraries or files that aren't modified (e.g. Bootstrap, jQuery, images, etc) inside `vendor.js` and they will be bundled together into their own HTML. Content hash remains the same, which allows the browser to cache them separately thereby reducing load time.

Reviewer 2: Is it HTML tho @montogeek WDYT?
```suggestion
__Why?__ With this you can import required libraries or files that aren't modified (e.g. Bootstrap, jQuery, images, etc) inside `vendor.js` and they will be bundled together into their own chunk. Content hash remains the same, which allows the browser to cache them separately thereby reducing load time.
```

Reviewer: Nope

Reviewer 2: So what is it, i thoughts its chunking or?

Reviewer: chunk is fine
